php/class-photonfill.php

```diff
@@ -603,6 +603,18 @@ public function get_attachment_image( $attachment_id, $size = 'full', $attr ) {
 		 * Get a lazy loaded img element
 		 */
 		public function get_lazyload_image( $attachment_id, $size = 'full', $attr = array() ) {
+			$srcset = $this->get_responsive_image_attribute( $attachment_id, $size, 'data-srcset' );
+			$sources = explode( ',', $srcset );
```

Comment:
if we were going to keep the wp_get_attachment_img these lines aren't necessary. But you will probably need these when you refactor.